lib/prepare_security.js

```diff
@@ -0,0 +1,207 @@
+import nv from '@pkgjs/nv';
+import auth from './auth.js';
+import Request from './request.js';
+
+const TEMPLATE = `
+## Planning
+
+* [X] Open an [issue](https://github.com/nodejs-private/node-private) titled
```

Comment:
It seems the template is static, would it make more sense as a separate `.md` file?